pipelines/pipeline_katello_36_to_37_upgrade.yml

```diff
@@ -0,0 +1,106 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-katello-3.6-3.7
+    forklift_boxes:
+      pipeline-katello-3.6-3.7-centos7:
+        box: centos7
+        memory: 6144
+      pipeline-proxy-3.6-3.7-centos7:
+        box: centos7
+        memory: 3072
+  roles:
+    - forklift
+
+- hosts:
+  - pipeline-katello-3.6-3.7-centos7
+  - pipeline-proxy-3.6-3.7-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 5
```

Comment:
I'd encourage you to test the puppet upgrade rather than starting with P5.

Reply:
take a look ... I think I've approximated what you did with 3.5